web/thesauruses/python/control_structures.json

```diff
@@ -39,10 +40,13 @@
       "code": "expression_if_true if condition else expression_if_false"
     },
     "while_loop": {
-      "code": "while condition:\n    statements"
+      "code": "while condition:\n    statements\n\nwhile True:\n    statements\n    if(conditional):\n        break"
     },
     "foreach_loop": {
-      "code": "for iterator in iterable:\n    statements"
+      "code": "for iterator in iterable:\n    statements\n\nfor first, second in iterable_of_tuples:\n    foo(first)\n    bar(second)"
+    },
+    "list_comprehension": {
+      "code": "[foo(item) for item in iterable]\n[foo(item) for item in iterable if conditional(item)]"
```

Comment:
Why do you have `[` and `]` around these? You don't use those in Python's list comprehensions. The code should be technically compilable.